src/coinbase/wallet.ts

```diff
@@ -89,6 +89,28 @@ export class Wallet {
     return Wallet.init(response.data!, "");
   }
 
+  /**
+   * Imports a Wallet for the given Wallet data.
+   *
+   * @param data - The Wallet data to import.
```

Comment:
nit: add params for `data.walletId` and `data.seed`. Also in doc string for user class `importWallet`